script/brave_license_helper.py

```diff
@@ -29,6 +29,10 @@ def AddBraveCredits(prune_paths, special_cases, prune_dirs, additional_paths):
         os.path.join('brave', 'third_party', 'android_deps'),
         os.path.join('brave', 'third_party', 'blink'),
         os.path.join('brave', 'third_party', 'libaddressinput'),
+
+        # misc tools
```

Comment:
The reason we can ignore `depot_tools` is that it's under the same license as Chromium and it's pretty part of the Chromium project, which we already credit elsewhere. So how about this as a comment:
```
# Part of the Chromium project and licensed under the same license as the rest of the project.
```

Reply:
it seemed similar to https://github.com/brave/brave-core/pull/6278#discussion_r463844747 because this is just a tool we use, none of the code ends up in Brave

Reply:
basically these are similar to devDependencies we have in package.json that don't get checked because the code isn't included in the browser

Reply:
In that case, let's exclude them with the following comment:
```
# These build dependencies don't end up in the binaries.
```